contractcourt/channel_arbitrator.go

```diff
@@ -1362,6 +1369,11 @@ func (c *ChannelArbitrator) sweepAnchors(anchors *lnwallet.AnchorResolutions,
 			c.cfg.Budget.AnchorCPFP,
 		)
 
+		// The anchor output in itself has a small output value of 330
+		// sats so we also include it in the budget to pay for the
+		// cpfp transaction.
+		budget += AnchorOutputValue
```

Comment:
nit: rather than doing an increment reassignment, just assign budget once by doing `budget := calculateBudget(...) + AnchorOutputValue`.

Not blocking.

Reply:
definitely the way to go